src/app/pages/view-test-result/components/vehicle-details-card/vehicle-details-card.html

```diff
@@ -31,7 +31,11 @@ <h2 class="des-header-style-4" id="view-test-result-vehicle-details-title">Vehic
       <data-row-custom
         *ngIf="vehicleDetails"
         [label]="'Vehicle details'"
-        [shouldHaveSeperator]="!!instructorRegistrationNumber || shouldShowDimensions"
+        [shouldHaveSeperator]="!!(instructorRegistrationNumber
+         || shouldShowDimensions
+         || schoolBike
+         || (!isADI3() && trainerPRN)
```

Comment:
suggest we create a function to handle this as its getting rather messy, that way we can write some unit tests to cover all the desired scenario's